db/sql/cassandra/connection.go

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) 2017 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cassandra
+
+import (
+	"strings"
+	"time"
+
+	"github.com/gocql/gocql"
+)
+
+// Config Configuration for Cassandra clients loaded from a configuration file
+type Config struct {
+
+	// A list of host addresses of cluster nodes.
+	Hosts []string `json:"hosts"`
+
+	// port for Cassandra (default: 9042)
+	Port int `json:"port"`
```

Comment:
Could you make this consistent with etcdv3 & redis configs?

endpoints []string